build.gradle

```diff
@@ -532,6 +532,16 @@ def executableAndArg = System.getProperty('os.name').toLowerCase().contains('win
 task distDocker  {
   dependsOn dockerDistUntar
   def dockerBuildVersion = 'develop'
+
+  if (project.hasProperty('includeCommitHash') && project.property('includeCommitHash').toBoolean()) {
+    def commitHash = new ByteArrayOutputStream()
+    exec {
+      commandLine 'git', 'rev-parse', '--short', 'HEAD'
+      standardOutput = commitHash
+    }
+    dockerBuildVersion = "master-" + commitHash
```

Comment:
I think we should do `dockerBuildVersion += "-" + commitHash` since it's technically develop-{commit_hash}

Reply:
Yeah I hesitated about it because we don't really have a develop branch... We probably should rename master to develop one day

Reply:
I renamed it to `develop-commitHash`